eng/common/pipelines/templates/steps/save-package-properties.yml

```diff
@@ -0,0 +1,43 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: DiffDirectory
+    type: string
+    default: $(Build.ArtifactStagingDirectory)/diff
+  - name: PackageInfoDirectory
+    type: string
+    default: $(Build.ArtifactStagingDirectory)/PackageInfo
+  - name: TargetPath
+    type: string
+    default: $(Build.SourcesDirectory)
+
+steps:
+  - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+      - task: Powershell@2
```

Comment:
Nit: too much indentation

Reply:
It's the default for the formatter.   That's 2 spaces inside of the first object property

Reply:
I prefer flat sequences as opposed to indented.  Sequences in sequences make it really obvious:

```yaml
steps:
- ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
  - task: Powershell@2
```